main.tf

```diff
@@ -10,3 +10,16 @@ resource "azurerm_role_assignment" "aks_network" {
   role_definition_name = "Network Contributor"
   principal_id         = azurerm_kubernetes_cluster.aks.identity[0].principal_id
 }
+
+module "flyte" {
```

Comment:
Agreed.  I think having the root module call the child module simplifies the platform-apps configuration a bit. The root module already has all the required information to configure Flyte, avoiding the need to explicitly set them in platform-apps.  This would help reduce coupling between the terraform-azure-aks and platform-apps repos.

Reply:
At a high level the instantiation of the `flyte` module needs to be consistent with eks and be done on platform apps, regardless of the flavor composition(multiple vs monolithic modules). Other than living in the same repo the `main` part of the root module should be disassociated from the `flyte` module.